css/home.css

```diff
@@ -69,21 +70,32 @@ body {
 }
 
 .homepage-hero__article {
-  margin-right: 5%;
-  margin-left: 5%;
+  margin: auto;
+}
+
+.homepage-hero__hdlp-logo{
```

Comment:
what is HDLP?

Reply:
"Hostal donde las paisas"

Reply:
Maybe this should be `.homepage-hero__logo` and if you have a particular style for the Hostal logo, you should create other class with a modifier `.homepage-hero__logo--hdlp`

Reply:
+45262847 @camigomez35

Reply:
Yeah actually main-nav's was named main-nav__logo.